City-of-Helsinki/drupal-helfi-platform-config
# UHF-1476 Remove service unit nodes

How to test: 
If you DO NOT have a running platform
- `composer create-project City-of-Helsinki/drupal-helfi-platform:dev-main hel-platform --no-interaction --repository https://repository.drupal.hel.ninja/`
- `cd hel-platform`
- Switch to corresponding branches: `composer require drupal/helfi_platform_config:dev-UHF-1476_remove_service_unit_nodes`
- `make new`
- Log in to the site and check that there is no Unit or Service content type or test content for them. 

If you have a running setup:
- Switch to corresponding branches: `composer require drupal/helfi_platform_config:dev-UHF-1476_remove_service_unit_nodes`
- `make drush-cr drush-updb`
- Log in to the site and check that there is no Unit or Service content type or test content for them. 

Also check these:
https://github.com/City-of-Helsinki/drupal-helfi/pull/103
https://github.com/City-of-Helsinki/drupal-helfi-kymp/pull/4
https://github.com/City-of-Helsinki/drupal-helfi-sote/pull/6
